conda_smithy/cli.py

```diff
@@ -596,13 +596,22 @@ def __call__(self, args):
                 all_good = False
                 print(
                     "{} has some lint:\n  {}".format(
-                        recipe, "\n  ".join(lints)
+                        recipe,
+                        "\n  ".join(
```

Comment:
could be:
```py
from textwrap import indent
# ...
indent("\n".join(lints), " " * 4)
```
et.c

Reply:
Or even:

```python
print(f"{recipe} has some lint:", *lints, sep="\n  ")
```

Reply:
None of the above suggestions does what I do in this PR. i.e. all lint need two space indent and each multi-line lint need 4 spaces for the second line and after.